skiko/src/jvmMain/cpp/common/Image.cc

```diff
@@ -47,6 +47,8 @@ extern "C" JNIEXPORT jlong JNICALL Java_org_jetbrains_skia_ImageKt__1nMakeFromPi
 
 extern "C" JNIEXPORT jlong JNICALL Java_org_jetbrains_skia_ImageKt__1nMakeFromEncoded
   (JNIEnv* env, jclass jclass, jbyteArray encodedArray, jint encodedLen) {
+    if (!encodedArray) return 0;
```

Comment:
Instead of returning 0 and throwing an exception, better to make `bytes` not-null:
```
fun makeFromEncoded(bytes: ByteArray): Image
```
Then the error will be in compile-time instead of runtime.

(we still don't support backward compatibility between Skiko versions, so this change will be okay)